Util/include/Poco/Util/PropertyListConfiguration.h

```diff
@@ -0,0 +1,94 @@
+//
+// PropertyListConfiguration.h
+//
+// Library: Util
+// Package: Configuration
+// Module:  PropertyListConfiguration
+//
+// Definition of the PropertyListConfiguration class.
+//
+// Copyright (c) 2017-2018, Applied Informatics Software Engineering GmbH.
+// and Contributors.
+//
+// SPDX-License-Identifier:	BSL-1.0
+//
+
+
+#ifndef Util_PropertyListConfiguration_INCLUDED
+#define Util_PropertyListConfiguration_INCLUDED
+
+
+#include "Poco/Util/Util.h"
+
+
+#ifndef POCO_UTIL_NO_XMLplistCONFIGURATION
```

Comment:
where is this define coming from?
this should probably be simply #ifndef POCO_UTIL_NO_XML_CONFIGURATION, just like the original XMLConfiguration, plus there should be an ifdef so this code only comes in on macOS

Reply:
Ok, yes it should be POCO_UTIL_NO_XML_CONFIGURATION since it does depend on XML.  It doesn't depend on any OS X code, so I think it should be available for all platforms like ini, json, xml, etc.